src/unix/bash.js

```diff
@@ -67,11 +64,12 @@ export function getEscapeFunction(options) {
 /**
  * Quotes an argument for use in Bash.
  *
- * @param {string} arg The argument to quote.
- * @returns {string} The quoted argument.
+ * @param {string} arg The argument to quote and escape.
+ * @returns {string} The quoted and escaped argument.
  */
 function quoteArg(arg) {
-  return `'${arg}'`;
+  const escapedArg = escapeForQuoted(arg);
```

Comment:
_(This comment applies to all similar changes in other shell-specific modules.)_

Can we, and would it make sense to, inline the function `escapeForQuoted` into `quoteArg`?

Reply:
Resolved with 4254aaa3d479fc5735a44a885aa92536ecac327f